build_scripts/build_usd.py

```diff
@@ -393,7 +393,10 @@ def RunCMake(context, force, extraArgs = None):
 
     # Note - don't want to add -A (architecture flag) if generator is, ie, Ninja
     if IsVisualStudio2019OrGreater() and "Visual Studio" in generator:
-        generator = generator + " -A x64"
+        if "ARM" in os.environ.get('PROCESSOR_IDENTIFIER'):
```

Comment:
Could the `platform` module be used here?
https://docs.python.org/3/library/platform.html#module-platform

Reply:
It could be, but problems arise when running an emulated x64 version of python (which is easy to do by mistake, as that's what the website gives you by default).

This way checks the name of the processor directly, which is not affected by emulation.

Reply:
It could be, but problems arise when running an emulated x64 version of python (which is easy to do by mistake, as that's what the website gives you by default).

This way checks the name of the processor directly, which is not affected by emulation.